src/main/cpp/tests/shuffle_split.cu

```diff
@@ -396,6 +396,22 @@ TEST_F(ShuffleSplitTests, ShortNulls)
     run_split(tbl, {2, 5});
     run_split(tbl, {0, 2, 4});
   }
+
+  // 34 rows (two input words) split such that the last two rows of validity are written as overflow
+  // bits in the copy kernel.
+  // - split at row 22, causing the second copy to have 12 bits to write.
+  // - the first 10 of those 12 bits goes in the first word, the last 2 bits goes into the second
+  // word. this is an
```

Comment:
Is there an inadvertent new-line in the comments here?

Reply:
Yeah, this is what pre-commit did.  🤷